ASSETS.md

```diff
@@ -66,6 +66,31 @@ The definition of the manifest is not complete yet and further details will foll
 
 Question: How do we handle if only one metricset is in beta but integration is in GA?
 
+### changelog.yml
+
+The changelog of a package contains always all previous changes and not only the one from the last major, minor, bugfix release. 
+Each array entry is a release. The type entry can contain the following values: [added, bugfix, deprecated, breaking-change, known-issue]
+
+The file looks as following:
+
+```
+- version: 1.0.4
+  changes:
+    - description: >
```

Comment:
@hbharding Based on the designs, we might also need a title + description. Would be great to have an icon for each "type" of change, but not a requirement.